src/test/cypress/package.json

```
@@ -10,6 +10,7 @@
         "@4tw/cypress-drag-drop": "2.0.0",
         "@types/node": "16.9.4",
         "cypress": "8.4.1",
+        "cypress-wait-until": "^1.7.1",
```

Use a fixed version to avoid sudden issues by automatic updates in yarn install
```suggestion
        "cypress-wait-until": "1.7.1",
```